app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -1,25 +1,29 @@
-- if @logs.blank? && @context != 'calendar'
-  = content_tag(:div, t('common.no_records'), class: 'no-records-msg')
+- @context = @context ? @context : 'logs_list'
+- step_modal_context = @context == ('calendar' || 'timeline') ? true : false
+
+- if @logs.blank?
+  - if step_modal_context
+    = content_tag(:div, t('common.no_records'), class: 'no-records-msg')
 - else
-  .show-actions.hidden-print class=(@context == 'calendar' ? 'nested-expander' : '')
-    - if @context == 'calendar'
+  .show-actions.hidden-print class=(step_modal_context ? 'nested-expander' : '')
+    - if step_modal_context
       a data-expands="step-logs-#{@step.id}" = t('log.show_logs_n', n: @step.logs_count)
       a data-hides="step-logs-#{@step.id}" = t('log.hide_logs_n', n: @step.logs_count)
     - unless @context == 'dashboard'
       a data-expand-all=".logs-index" href="#" = t('common.expand_all')
       a data-hide-all=".logs-index" href="#"
-        = @context == 'calendar' ? t('common.condense_all') : t('common.hide_all')
+        = step_modal_context ? t('common.condense_all') : t('common.hide_all')
 
-  - unless @context == 'calendar'
-    - if @step
-      h5
-        - if @step.milestone?
-          i.fa.fa-fw.fa-flag-o
-        - elsif @step.checkin?
-          i.fa.fa-fw.fa-calendar
-        = @step.summary
+  / TODO: Under what context is the large heading for step needed?
```

Comment:
This is a styled choice to show a step heading.

However, later down in the logic, you'll see that the step name already is displayed as link in the step. I'm unsure why we had this logic previously. We are only using the linked step name at this time.